scripts/standardise_validate.py

```diff
@@ -40,12 +85,31 @@ def main() -> None:
     if is_argo():
         concurrency = 4
 
+    # The target exists, consider this is a resumed workflow
+    is_resuming = exists(arguments.target)
```

Comment:
This might be not helpfull as if one task did create at least one file, thie target directory will exists so all new instance of the script will be in a "resume" mode

Reply:
I think this may be better handled in the `run_standardising` where each standardizing checks to see if the file exists before processing rather than trying to figure out what needs to process first.